db/schema.rb

```diff
@@ -32,9 +32,11 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "role"
+    t.integer  "user_id"
```

Comment:
every table has an `id` column (known as the "primary key"), which does not need to be indexed.  A `user_id` looks more like a "foreign key" which would be on another table referencing the users table.  For example, in the last project, the Posts table had a user_id column which is how a post was associated to (or belongs_to) a user.  In this case, it wouldn't make much sense for a user to need a foreign key associated to itself (a user belongs_to a user), so I don't think this is the column name you should be using.

What are you trying to accomplish here?